### ISSUE TITLE
Astro C40 controller is supported but doesnt show up.

### ISSUE THREAD

**Issue by DarkRqlphYT** [Ryochan7/DS4Windows#3215]
**Describe the bug**
In the supported website list the astro c40 is working but mine isnt detecting.

**To Reproduce**??
Steps to reproduce the behavior:???

**Expected behavior**
To see the controller

**Screenshots and Logs**
If applicable, add screenshots to help explain your problem. Also, please add the most
log file (ds4windows_log.txt) from the Logs folder in your DS4Windows config folder.
![Capture](https://github.com/Ryochan7/DS4Windows/assets/71142305/07537447-06eb-44db-a69f-0f8c56d7e03c)
[ds4windows_log.txt](https://github.com/Ryochan7/DS4Windows/files/13775043/ds4windows_log.txt)

**Desktop (please complete the following information):**
 - Controller Make and Model: Astro C40    ????(CUH-ZCT2U)?????
 - OS: Microsoft Windows NT 10.0.19045.0
 - DS4Windows Version 3.3.2

**Additional context**
![Capture1](https://github.com/Ryochan7/DS4Windows/assets/71142305/655fa5f6-29ec-4584-9110-f03ddc9db4a7)

**Comment by tmd02-cpu** [comment]
I found on Reddit that the Astro C40 thinks it's an xbox controller on Windows 10, and even their dedicated software forces it into PC mode, this happened to me but the steps I followed got it recognized by DS4Windows again.

Open Registry Editor on Windows (search in start menu) and navigate to: "Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Enum\USB\VID_9886&PID_0025"

It can be confusing looking at all the sub-catergories on the left side of Registry Editor, once you find this find values named "LowerFilters" with data "logi_joy_usb_lo".

Right-click "LowerFilters" and rename it to "xxx_LowerFilters". the Astro will now think it's in PS4 mode and DS4Windows will recognize he controller. You will have to revert the name back to what it was in order to use the calibration software for the Astro C40. I'll find the original link to show you if I've missed anything.
